### PR TITLE
Fix issue 27093: Explain :focus-visible example

### LINKED ISSUE #27093: No difference between mouse and keyboard focus methods
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible

### What specific section or headline is this issue about?

https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible#basic_example

### What information was incorrect, unhelpful, or incomplete?

There is supposed to be a difference when putting focus on an element through the keyboard versus the mouse focus method. This difference is not visible in current browsers. Tested in Chrome and Firefox.

### What did you expect to see?

I expected to see a difference, because this was indicated in the text.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

"Compare what happens when you click on the different controls with a mouse, versus when you tab through them using a keyboard. "

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/css/_colon_focus-visible`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/_colon_focus-visible/index.md
* Last commit: https://github.com/mdn/content/commit/5fea7c9593f5e4b4ef13ec65064acf1eabf01e4e
* Document last modified: 2023-02-21T06:49:52.000Z

</details>

### PR DESCRIPTION
**Pull request by wbamberg**
Fixes https://github.com/mdn/content/issues/27093.

I am very tempted to remove the text controls from the example, because they don't show any difference, and I haven't talked about them as I felt the explanation was already quite long. We do already talk about the difference between different controls in the body of the page.

### CODE REVIEW

File: files/en-us/web/css/_colon_focus-visible/index.md [hunk 1]
```diff
@@ -61,7 +67,13 @@ button {
 }
 ```
 
-{{EmbedLiveSample("Basic_example", "100%", 300)}}
+If you click each element in turn, you will see that when `:focus` is used to style the focus ring, the UA draws the focus ring when the user clicks the button. However, when `:focus-visible` is used to style the focus ring, the UA does not draw the focus ring when the user clicks the button, just like in the default case.
+
+If you then tab through each element, you will see that in all three cases — default, `:focus`, `:focus-visible` — the UA draws the focus ring around the button when the user navigates to it with the keyboard.
```
Comment (estelle) [inline files/en-us/web/css/_colon_focus-visible/index.md:72]:
```suggestion
If you then tab through each element, you will see that in all three cases — default, `:focus`, and `:focus-visible` — the UA draws the focus ring around the button when the user navigates to it with the keyboard.
```

i might be wrong here... I assume the and is always needed, but there may be a rule of not needing it when using m-dashes for a clause like this.

**Review by estelle — APPROVED** [review]
Not merging because there is one "maybe edit"

**Comment by estelle** [comment]
Definitely leave the two types of controls. Users don't necessarily know that inputs get a focus ring too.

File: files/en-us/web/css/_colon_focus-visible/index.md (hunk 1 above)
Comment (wbamberg) [reply files/en-us/web/css/_colon_focus-visible/index.md:72]: yeah, I like the "and" here.

**Pull request merged**